### PR TITLE
Improvements to Others tab on "no data" screen

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Updated design for Other ways tab, #PG-2871

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/SitesManager/lang/en.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
         "MainDescription": "Your Web Analytics reports need Websites! Add, update, delete Websites, and show the JavaScript to insert in your pages.",
         "MenuManage": "Manage",
         "MobileAppsAndSDKs": "Mobile apps and SDKs",
+        "MobileAppsAndSDKsModified": "Mobile Apps and SDKs",
```
Comment (sgiehl) [inline plugins/SitesManager/lang/en.json]: Only changing the writing case of `apps`, doesn't require to change the key.

Comment (AltamashShaikh) [reply plugins/SitesManager/lang/en.json]: Thanks :+1:

File: plugins/CoreAdminHome/templates/trackingCodeGenerator.twig [hunk 2]
```diff
@@ -20,7 +20,7 @@
             {{ 'General_GoTo2'|translate }}:
             <a href="#/javaScriptTracking">{{ 'CoreAdminHome_JavaScriptTracking'|translate  }}</a>
             <a href="#/imageTracking">{{ 'CoreAdminHome_ImageTracking'|translate }}</a>
-            <a href="#/importServerLogs">{{ 'CoreAdminHome_ImportingServerLogs'|translate }}</a>
```
Comment (sgiehl) [inline plugins/CoreAdminHome/templates/trackingCodeGenerator.twig:23]: The translation key `CoreAdminHome_ImportingServerLogs` now seems no longer to be used, so we can remove it from the (english) translation file.

Comment (AltamashShaikh) [reply plugins/CoreAdminHome/templates/trackingCodeGenerator.twig:23]: done

**Review by sgiehl — APPROVED** [review]

**Pull request merged**